internal/ledger/account.go

```diff
@@ -52,8 +52,13 @@ func (o *Account) GetState(key []byte) (bool, []byte) {
 	}
 
 	val, err := o.ldb.Get(append(o.Addr.Bytes(), key...))
-	if err != nil && err != errors.ErrNotFound {
-		panic(err)
+	if err != nil {
```

Comment:
Extract a method to deal with this error

Reply:
Yes, will refactor storage to deal with errors.ErrNotFound in another PR.

Reply:
LGTM